distributed/bokeh/worker.py

```diff
@@ -421,6 +421,40 @@ def update(self):
 
             self.last = monitor.count
 
+class Counters(DashboardComponent):
+    def __init__(self, server, sizing_mode='scale_width', **kwargs):
+        self.server = server
+        self.figures = {}
+        self.sources = {}
+        self.sizing_mode = sizing_mode
+
+        figures = [self.add_figure(name)
+                   for name in sorted(self.server.counters)]
+
+        self.root = column(figures, sizing_mode=sizing_mode)
+
+    def add_figure(self, name):
+        source = ColumnDataSource({'x': [], 'y': []})
+        fig = figure(title=name, tools='', height=150, sizing_mode=self.sizing_mode)
+        fig.line(source=source, x='x', y='y')
+        self.sources[name] = source
+        self.figures[name] = fig
+        fig.yaxis.visible = False
+        return fig
+
+    def update(self):
+        for name, figure in self.figures.items():
+            counter = self.server.counters[name]
+            xs = [counter.quantile(i / 100) for i in range(1, 100)]
+            try:
+                ys = [1 / (xs[i + 1] - xs[i]) for i in range(len(xs) - 1)]
```

Comment:
I'm not sure that this is the right way to get a sense for the distribution visually out of the quantiles.  Recommendations welcome here.  In particular this fails when two centroids overlap.